test/jsApi.yml

```diff
@@ -0,0 +1,130 @@
+- test: Test js input
+  steps:
+    -   in: bn create node8 returnInput
+    -   in: |-
+            echo 'exports.handler = (input) => ({ inputType: typeof input, input});' >function.js
+    -   in: bn deploy returnInput
+    -   in: curl -s "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{}}'
+    -   in: curl -s --data '{"' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"undefined"}'
+    -   in: curl -s --data '{}' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{}}'
+    -   in: curl -s --data '{"foo":"bar","yet":123}' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":{"foo":"bar","yet":123}}'
+    -   in: curl -s --data '["foo","bar"]' "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnInput"
+        out: '{"inputType":"object","input":["foo","bar"]}'
+    -   in: bn remove returnInput
+- test: Test js method 
+  steps:
+    -   in: bn create node8 returnMethod
+    -   in: |-
+            echo 'exports.handler = (_, req) => req.method;' >function.js
+    -   in: bn deploy returnMethod
+    -   in: curl -s -X GET "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"GET"'
+#    -   in: curl -s -X HEAD "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+#        out: '"HEAD"'
+    -   in: curl -s -X POST "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"POST"'
+    -   in: curl -s -X PUT "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"PUT"'
+    -   in: curl -s -X DELETE "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"DELETE"'
+#    -   in: curl -s -X CONNECT "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+#        out: '"CONNECT"'
+    -   in: curl -s -X OPTIONS "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"OPTIONS"'
+#    -   in: curl -s -X TRACE "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+#        out: '"TRACE"'
+    -   in: curl -s -X PATCH "https://${BINARIS_INVOKE_ENDPOINT}/v1/run/${BINARIS_API_KEY}/returnMethod"
+        out: '"PATCH"'
+    -   in: bn remove returnMethod
+- test: Test js data input
```

Comment:
Nit: (but please fix) I really prefer a newline in between different test blocks. It's much easier to read when they are separated.